membership-common/src/main/scala/com/gu/memsub/subsv2/reads/CatJsonReads.scala

```diff
@@ -1,91 +1,64 @@
 package com.gu.memsub.subsv2.reads
```

Comment:
changes in this file are mostly just tidying up fields that we no longer need and renaming to use more standard zuora terminology